snoop/data/exportimport.py

```diff
@@ -27,172 +25,15 @@
-    )
-
-    emlx_reconstruct_tasks = (
-        models.Task.objects
-        .filter(func='emlx.reconstruct')
-        .annotate(arg0=RawSQL('(args ->> 0)::integer', ()))
-        .filter(arg0__in=files)
-    )
-
-    exif_extract_tasks = (
-        models.Task.objects
-        .filter(func='exif.extract')
-        .filter(blob_arg__in=file_blob_pks)
-    )
-
-    filesystem_create_archive_files_tasks = (
-        models.Task.objects
-        .filter(func='filesystem.create_archive_files')
-        .annotate(arg0=RawSQL('(args ->> 0)::integer', ()))
-        .filter(arg0__in=files)
-    )
-
-    filesystem_create_attachment_files_tasks = (
-        models.Task.objects
-        .filter(func='filesystem.create_attachment_files')
-        .annotate(arg0=RawSQL('(args ->> 0)::integer', ()))
-        .filter(arg0__in=files)
-    )
-
-    filesystem_handle_file_tasks = (
-        models.Task.objects
-        .filter(func='filesystem.handle_file')
-        .annotate(arg0=RawSQL('(args ->> 0)::integer', ()))
-        .filter(arg0__in=files)
-    )
-
-    filesystem_walk_tasks = (
-        models.Task.objects
-        .filter(func='filesystem.walk')
-        .annotate(arg0=RawSQL('(args ->> 0)::integer', ()))
-        .filter(arg0__in=directories)
-    )
-
-    tika_rmeta_email_tasks = (
-        models.Task.objects.filter(next_set__in=(
-            models.TaskDependency.objects
-            .filter(next__in=email_parse_tasks)
-        ))
-        .filter(func='tika.rmeta')
-    )
-    tika_rmeta_file_tasks = (
-        models.Task.objects
-        .filter(func='tika.rmeta')
-        .filter(blob_arg__in=file_blob_pks)
-    )
-
-    tika_rmeta_tasks = (
-        models.Task.objects
-        .filter(
-            Q(pk__in=tika_rmeta_email_tasks) |
-            Q(pk__in=tika_rmeta_file_tasks)
-        )
-    )
-
-    tasks = (
-        models.Task.objects
-        .filter(
-            Q(pk__in=archives_unarchive_tasks) |
-            Q(pk__in=digests_gather_tasks) |
-            Q(pk__in=digests_index_tasks) |
-            Q(pk__in=digests_launch_tasks) |
-            Q(pk__in=email_msg_to_eml_tasks) |
-            Q(pk__in=email_parse_tasks) |
-            Q(pk__in=emlx_reconstruct_tasks) |
-            Q(pk__in=exif_extract_tasks) |
-            Q(pk__in=filesystem_create_archive_files_tasks) |
-            Q(pk__in=filesystem_create_attachment_files_tasks) |
-            Q(pk__in=filesystem_handle_file_tasks) |
-            Q(pk__in=filesystem_walk_tasks) |
-            Q(pk__in=tika_rmeta_tasks)
-        )
-    )
-
-    blobs = (
-        models.Blob.objects
-        .filter(
-            Q(pk__in=file_original_pks) |
-            Q(pk__in=file_blob_pks) |
-            Q(pk__in=tasks.values_list('result_id', flat=True)) |
-            Q(pk__in=tasks.values_list('blob_arg_id', flat=True))
-        )
-    )
-
-    task_dependencies = (
-        models.TaskDependency.objects
-        .filter(next__in=tasks)
-    )
+def build_export_queries():
```

Comment:
This is not needed anymore. The database export will be done by copying the db directory.